### PR TITLE
 Update MockEnvironmentBuilder.ts

### PR DESCRIPTION
**Pull request by jfallaire**

### CODE REVIEW

File: src/MockEnvironment/MockEnvironmentBuilder.ts [hunk 1]
```diff
@@ -125,6 +125,8 @@ export class MockEnvironmentBuilder {
     this.searchInterface.queryStateModel = this.queryStateModel;
     this.searchInterface.componentStateModel = this.componentStateModel;
     this.searchInterface.componentOptionsModel = this.componentOptionsModel;
+    this.searchInterface.getBindings = () => this.getBindings() as any;
```
Comment (francoislg) [inline src/MockEnvironment/MockEnvironmentBuilder.ts:128]: 🤔 why is this cast to `any` ?

**Review by francoislg — APPROVED** [review]
👍 Cool contribution, thanks :)

File: src/MockEnvironment/MockEnvironmentBuilder.ts (hunk 1 above)
Comment (jfallaire) [reply src/MockEnvironment/MockEnvironmentBuilder.ts:128]: Good point, we don't need to cast to `any`

**Comment by jfallaire** [comment]
Great! this change will be useful for our PS Project generator which is now using this `search-ui-tests`

**Pull request merged**